test/e2e/tests/navbar/appearanceMenu.spec.ts

```diff
@@ -0,0 +1,23 @@
+import {expect} from '@playwright/test'
+import {test} from '@sanity/test'
+
+test('color scheme changes and persists', async ({page, baseURL}) => {
+  await page.goto(baseURL ?? '/test/content')
+  await page.locator(`[id='user-menu']`).click()
+  await page.getByLabel('Use dark appearance').click()
+
+  const localStorage = await page.evaluate(() => window.localStorage)
+  expect(localStorage['sanityStudio:ui:colorScheme']).toBe('dark')
```

Comment:
small nit: maybe it might make sense to make a constant for the key above the file so when it changes only have to change one place